qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/AbstractTextDocumentService.java

```diff
@@ -167,6 +169,11 @@ public CompletableFuture<List<InlayHint>> inlayHint(InlayHintParams params) {
 		return CompletableFuture.completedFuture(null);
 	}
 
+	@Override
+	public CompletableFuture<WorkspaceEdit> rename(RenameParams params) {
+		return CompletableFuture.completedFuture(null);

```

Comment:
If you wish to use this stragegy which implement method in AbstractTextDocumentService.java, please do that for other methods (like inlayHint).

Reply:
I think it's already implemented this way

Reply:
Good catch, it is perfect.